xlive/Blam/Engine/simulation/simulation_entity_update_encode_helper.cpp

```diff
@@ -0,0 +1,78 @@
+#include "stdafx.h"
+#include "simulation_entity_update_encode_helper.h"
+
+bool c_entity_update_encode_helper::setup(c_bitstream* bitstream, uint32 required_leave_space_bits, uint32 update_component_first_index, uint32 update_component_count, uint32 update_mask)
+{
+	this->m_update_considered_mask = 0;
```

Comment:
remove `this->` spam in this file and just reference the `m_` members

Reply:
`m_` gives you all the intent you need.